seqr/views/utils/dataset_utils.py

```diff
@@ -449,3 +445,47 @@ def _load_rna_seq(model_cls, file_path, user, mapping_file, ignore_extra_samples
         logger.warning(warning, user)
 
     return samples_to_load, info, warnings
+
+
+PHENOTYPE_PRIORITIZATION_HEADER = ['tool', 'project', 'sampleId', 'rank', 'geneId', 'diseaseId', 'diseaseName']
+PHENOTYPE_PRIORITIZATION_REQUIRED_HEADER = PHENOTYPE_PRIORITIZATION_HEADER + ['scoreName1', 'score1']
+MAX_SCORES = 100
+
+
+def _parse_phenotype_pri_row(row):
+    record = {_to_snake_case(key): row.get(key) for key in PHENOTYPE_PRIORITIZATION_HEADER}
+
+    scores = {}
+    for i in range(1, MAX_SCORES):
+        score_name = row.get(f'scoreName{i}')
+        if not score_name:
+            break
+        scores[_to_camel_case(_to_snake_case(score_name))] = float(row[f'score{i}'])
```

Comment:
We have score fields like both `compositeLR` and `post_test_probability`. So I convert everything to snake case first, then to camel case.

Reply:
add a comment explaining this

Reply:
would it be easier to just store everything as snake case? Theres no real advantage to camel case...

Reply:
If we convert to snake case only, then `compositeLR` will become `composite_l_r`. The frontend function `snakecaseToTitlecase` will convert it to `Composite L R`. That is not what we want.